jenkins/Jenkinsfile

```diff
@@ -1,374 +1,496 @@
-                      sed -i "s/$IMAGE_BASE_NAME-$dp_version_formatted.*/$IMAGE_BASE_NAME-$dp_version_formatted-$imageID\\"/" $configFile; git add $configFile || true; git diff $configFile
+            steps {
+                script {
+                    for (String configFile: configPaths) {
+                        if (GENERATE_DATAPROC_IMAGE == "true") {
+                            for (String dp_version : DATAPROC_VERSIONS.trim().split(' ')) {
+                                def dp_version_formatted = dp_version.replace(".", "-")
+                                sh(
+                                    script: """
+                                        sed -i "s/$customDataprocImageBaseName-$dp_version_formatted.*/$customDataprocImageBaseName-$dp_version_formatted-$imageID\\"/" $configFile; git add $configFile || true; git diff $configFile
+                                    """
+                                )
+                            }
+                        }
+
+                        if (GENERATE_GCE_IMAGE == "true") {
+                            sh(
+                                script: """
+                                    sed -i "s/$customGceImageBaseName-.*/$customGceImageBaseName-$imageID\\"/" $configFile; git add $configFile || true; git diff $configFile
+                                """
+                            )
+                        }
+                    }
+                }
+
+                sshagent(['jenkins-ssh-github']) {
+                    sh """
+                        git checkout -b $branchName$imageID
+                        git commit -m "automated custom image hash updates for image ID $imageID"
+                        git push origin $branchName$imageID
                     """
-                )
-              }
-            }
-          }
+                }
 
-          sshagent(['jenkins-ssh-github']) {
-            sh """
-              git checkout -b $branchName$imageID
-              git commit -m "automated custom image hash updates"
-              git push origin $branchName$imageID
-            """
-          }
-
-        }
-      }
+            }
+    }
 
-      stage('Create PR') {
+    stage('Create PR') {
         when { expression { !shouldAbortBuild } }
-        steps {
-          script {
-            def changeDescription = getChangeDescription(versionMap)
-            withCredentials([file(credentialsId: 'hub-token', variable: 'token')]) {
-              sh """
-                docker pull us.gcr.io/broad-dsp-gcr-public/hub:1
+            steps {
+                script {
+                    def changeDescription = getChangeDescription(versionMap)
+                    withCredentials([file(credentialsId: 'hub-token', variable: 'token')]) {
+                    sh """
+                        docker pull us.gcr.io/broad-dsp-gcr-public/hub:1
 
-                docker rm -f hub-runtime | true
+                        docker rm -f hub-runtime || true
```

Comment:
I assume `|` was a typo that hasn't caused issues.

Reply:
I get them mixed up sometimes 😖

Reply:
How dare you...not like it's just one more tiny line  😛